Content.Server/Medical/HealingSystem.cs

```diff
@@ -237,39 +237,78 @@ private bool HasDamage(Entity<DamageableComponent> ent, HealingComponent healing
     private string? GetDamageGroupByType(string id)
         => (from @group in _prototypes.EnumeratePrototypes<DamageGroupPrototype>() where @group.DamageTypes.Contains(id) select @group.ID).FirstOrDefault();
 
-    private bool IsBodyDamaged(Entity<BodyComponent> target, EntityUid user, HealingComponent healing)
+
+    ////////////// GOOBEDIT Start
+    // Target: the target Entity
+    // User: The person trying to heal, can be null.
+    // healing: the healing component
+    // targetedPart: bypasses targeting system to specify a limb. Must be set if user is null.
```

Comment:
If you're going to add comments, follow the documentation scheme from other commenting structures, and don't put 10 million slashes. I.e. something like this.
```suggestion
    // Goobstation start
    /// <param name="xPosition">the new x-coordinate.</param>
    /// <param name="yPosition">the new y-coordinate.</param>
    [ . . . ]
```